### PR TITLE
Fix/missing imports

### PR DESCRIPTION
**Pull request by fartariatomas**
This pull will correct the missing imports and an error in the kudu tests.

### CODE REVIEW

File: kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala [hunk 1]
```diff
@@ -19,6 +19,14 @@ package com.datamountaineer.streamreactor.connect.cassandra.config
...
 import java.net.ConnectException
 
+import com.datamountaineer.connector.config.Config
+import com.datamountaineer.streamreactor.connect.cassandra.config.TimestampType.TimestampType
+import com.datamountaineer.streamreactor.connect.errors.{ErrorPolicy, ThrowErrorPolicy}
+import com.datastax.driver.core.ConsistencyLevel
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import org.apache.kafka.common.config.ConfigException
+import scala.collection.JavaConversions.asScalaIterator
+
```
Comment (goncaloccastro) [inline kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala]: This whitespace should be removed.

File: kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/Writer.scala [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+
```
Comment (goncaloccastro) [inline kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/Writer.scala]: This whitespace should be removed.

File: kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala (hunk 1 above)
Comment (fartariatomas) [reply kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala]: done

File: kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/Writer.scala (hunk 2 above)
Comment (fartariatomas) [reply kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/Writer.scala]: done

**Review by goncaloccastro — APPROVED** [review]

**Pull request merged**